oaknational/Oak-Web-Application
# feat: favicon generation build script

## Description

- adds build script to generate favicon set from logo which is fetched from sanity
- writes a JSON file `{ innerHTML: "<link ..." }` with all the favicon links inside
- uses [html-react-parser](https://www.npmjs.com/package/html-react-parser) to parse innerHTML inside _document/head
- **NB** it generates a lot of icons, [see the default (which we use) sources config here](https://github.com/itgalaxy/favicons#usage) 
- this will in theory allow the user to save the app to their home screen

## Issue(s)

Fixes #1388 

## How to test

1. Go to {cloud link} in all sorts of browsers
3. You should see the favicon in the tab (it's green now, as i couldn't find the blue version in our theme anywhere :thinking: )


## Screenshots

How it used to look (delete if n/a):

![image](https://user-images.githubusercontent.com/12934669/232788040-9b5a8aef-e525-49ad-99a9-8570080c51aa.png)

How it should now look:

![image](https://user-images.githubusercontent.com/12934669/232788008-559edb52-2d25-446e-8a6a-f8e50d66d11a.png)

## Checklist
